### ISSUE TITLE
 Can't install antergos : Cannot create download package list (metalinks) error: 1083445

### ISSUE THREAD

**Issue by dessalines** [Antergos/Cnchi#874]
Trying to install antergos on a new computer, I've tried a LOT of guides out there right now to fix this error, and none of them are working.

**Comment by dessalines** [comment]
```
2018-04-26 23:45:31 [DEBUG] select_packages.py(394) add_features_packages(): Selecting package chromium for feature chromium
2018-04-26 23:45:31 [DEBUG] select_packages.py(344) select_packages(): All features needed packages have been added
2018-04-26 23:45:31 [DEBUG] select_packages.py(361) select_packages(): Packages list: grub,antergos-gnome-meta,sudo,e2fsprogs,unrar,base-devel,net-tools,antergos-midnight-timers,btrfs-progs,efibootmgr,whois,alsa-utils,dmraid,tar,os-prober,python2-gnomekeyring,wireless_tools,ntfs-3g,python-pyqt5,acpid,exfat-utils,splix,screenfetch,htop,b43-fwcutter,gsfonts,nilfs-utils,mlocate,dosfstools,terminus-font,crda,base,cifs-utils,modemmanager,chromium,usb_modeswitch,bash-completion,bind-tools,ipw2100-fw,pkgfile,antergos-common-meta,hplip,f2fs-tools,wpa_actiond,unzip,nfs-utils,antergos-keyring,openssh,antergos-mirrorlist,grub2-theme-antergos,cups-pk-helper,reiserfsprogs,cups,wget,intel-ucode,reflector,xfsprogs,haveged,networkmanager,python-gobject,gzip,zd1211-firmware,ipw2200-fw,jfsutils,system-config-printer,antergos-desktop-settings
2018-04-26 23:45:31 [DEBUG] select_packages.py(111) create_package_list(): Packages selected
2018-04-26 23:45:31 [DEBUG] pac.py(122) initialize_alpm(): ALPM initialised with root dir / and db path /var/lib/pacman
2018-04-26 23:45:31 [DEBUG] pac.py(98) __init__(): ALPM repository database order is: ['antergos', 'core', 'extra', 'community', 'multilib']
2018-04-26 23:45:31 [ERROR] metalink.py(118) create(): Unable to create download queue for package grub. An exception of type TypeError occured. Arguments:
("'NoneType' object is not subscriptable",)
2018-04-26 23:45:31 [ERROR] download.py(150) create_metalinks_list(): Error creating metalink for package grub. Installation will stop
2018-04-26 23:45:31 [ERROR] download.py(182) create_metalinks_list(): Can't create download set. An exception of type InstallError occured. Arguments:
('Error creating metalink for package grub. Installation will stop',)
2018-04-26 23:45:31 [ERROR] process.py(130) run(): 'Cannot create download package list (metalinks).'
2018-04-26 23:45:31 [ERROR] process.py(135) run(): Traceback (most recent call last):
2018-04-26 23:45:31 [ERROR] process.py(135) run():   File "/usr/share/cnchi/src/installation/process.py", line 92, in run
    self.create_metalinks_list()
2018-04-26 23:45:31 [ERROR] process.py(135) run():   File "/usr/share/cnchi/src/installation/process.py", line 82, in create_metalinks_list
    raise misc.InstallError(txt)
2018-04-26 23:45:31 [ERROR] process.py(135) run(): misc.extra.InstallError: 'Cannot create download package list (metalinks).'
2018-04-26 23:45:32 [INFO] slides.py(276) manage_events_from_cb_queue(): Creating the list of packages to download...
2018-04-26 23:45:32 [DEBUG] connectionpool.py(208) _new_conn(): Starting new HTTP connection (1): build.antergos.com
2018-04-26 23:45:33 [DEBUG] connectionpool.py(396) _make_request(): http://build.antergos.com:80 "GET /hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=1083458&result=False HTTP/1.1" 301 None
2018-04-26 23:45:33 [DEBUG] connectionpool.py(396) _make_request(): http://build.antergos.com:80 "GET /api/hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=1083458&result=False HTTP/1.1" 200 None

(gst-plugin-scanner:2539): GStreamer-WARNING **: 23:45:35.572: Failed to load plugin '/usr/lib/gstreamer-1.0/libgstfluidsynthmidi.so': libfluidsynth.so.1: cannot open shared object file: No such file or directory

(gst-plugin-scanner:2539): GStreamer-WARNING **: 23:45:35.608: Failed to load plugin '/usr/lib/gstreamer-1.0/libgstkate.so': libkate.so.1: cannot open shared object file: No such file or directory

(gst-plugin-scanner:2539): GStreamer-WARNING **: 23:45:35.664: Failed to load plugin '/usr/lib/gstreamer-1.0/libgstlv2.so': liblilv-0.so.0: cannot open shared object file: No such file or directory

(gst-plugin-scanner:2539): GStreamer-WARNING **: 23:45:35.680: Failed to load plugin '/usr/lib/gstreamer-1.0/libgstopencv.so': libopencv_bgsegm.so.3.4: cannot open shared object file: No such file or directory

(gst-plugin-scanner:2539): GStreamer-WARNING **: 23:45:35.704: Failed to load plugin '/usr/lib/gstreamer-1.0/libgstqmlgl.so': libQt5Quick.so.5: cannot open shared object file: No such file or directory
error: XDG_RUNTIME_DIR not set in the environment.

(gst-plugin-scanner:2539): GStreamer-WARNING **: 23:45:36.281: Failed to load plugin '/usr/lib/gstreamer-1.0/libgstzbar.so': libzbar.so.0: cannot open shared object file: No such file or directory
1   0x7fd11311bdc8 /usr/lib/libjavascriptcoregtk-4.0.so.18(WTFCrash+0x18) [0x7fd11311bdc8]
2   0x7fd11728f7eb /usr/lib/libwebkit2gtk-4.0.so.37(+0x7817eb) [0x7fd11728f7eb]
3   0x7fd1172ca8c1 /usr/lib/libwebkit2gtk-4.0.so.37(+0x7bc8c1) [0x7fd1172ca8c1]
4   0x7fd1172cad58 /usr/lib/libwebkit2gtk-4.0.so.37(+0x7bcd58) [0x7fd1172cad58]
5   0x7fd1172cb411 /usr/lib/libwebkit2gtk-4.0.so.37(+0x7bd411) [0x7fd1172cb411]
6   0x7fd11818d90a /usr/lib/libwebkit2gtk-4.0.so.37(+0x167f90a) [0x7fd11818d90a]
7   0x7fd11818e236 /usr/lib/libwebkit2gtk-4.0.so.37(+0x1680236) [0x7fd11818e236]
8   0x7fd118181c85 /usr/lib/libwebkit2gtk-4.0.so.37(+0x1673c85) [0x7fd118181c85]
9   0x7fd11818a100 /usr/lib/libwebkit2gtk-4.0.so.37(+0x167c100) [0x7fd11818a100]
10  0x7fd118186b10 /usr/lib/libwebkit2gtk-4.0.so.37(+0x1678b10) [0x7fd118186b10]
11  0x7fd118193478 /usr/lib/libwebkit2gtk-4.0.so.37(+0x1685478) [0x7fd118193478]
12  0x7fd1181940fe /usr/lib/libwebkit2gtk-4.0.so.37(+0x16860fe) [0x7fd1181940fe]
13  0x7fd118181692 /usr/lib/libwebkit2gtk-4.0.so.37(+0x1673692) [0x7fd118181692]
14  0x7fd11819313b /usr/lib/libwebkit2gtk-4.0.so.37(+0x168513b) [0x7fd11819313b]
15  0x7fd118193241 /usr/lib/libwebkit2gtk-4.0.so.37(+0x1685241) [0x7fd118193241]
16  0x7fd1172c67c1 /usr/lib/libwebkit2gtk-4.0.so.37(+0x7b87c1) [0x7fd1172c67c1]
17  0x7fd1181c9674 /usr/lib/libwebkit2gtk-4.0.so.37(+0x16bb674) [0x7fd1181c9674]
18  0x7fd1181ce151 /usr/lib/libwebkit2gtk-4.0.so.37(+0x16c0151) [0x7fd1181ce151]
19  0x7fd1181cee29 /usr/lib/libwebkit2gtk-4.0.so.37(+0x16c0e29) [0x7fd1181cee29]
20  0x7fd117fd63ae /usr/lib/libwebkit2gtk-4.0.so.37(+0x14c83ae) [0x7fd117fd63ae]
21  0x7fd117fd6a49 /usr/lib/libwebkit2gtk-4.0.so.37(+0x14c8a49) [0x7fd117fd6a49]
22  0x7fd117e4c11e /usr/lib/libwebkit2gtk-4.0.so.37(+0x133e11e) [0x7fd117e4c11e]
23  0x7fd117e48f3f /usr/lib/libwebkit2gtk-4.0.so.37(+0x133af3f) [0x7fd117e48f3f]
24  0x7fd117e4f084 /usr/lib/libwebkit2gtk-4.0.so.37(+0x1341084) [0x7fd117e4f084]
25  0x7fd117737016 /usr/lib/libwebkit2gtk-4.0.so.37(+0xc29016) [0x7fd117737016]
26  0x7fd112e471ee /usr/lib/libjavascriptcoregtk-4.0.so.18(_ZN3JSC16callCustomSetterEPNS_9ExecStateEPFbS1_llEbNS_7JSValueES4_+0xe) [0x7fd112e471ee]
27  0x7fd112eeefbf /usr/lib/libjavascriptcoregtk-4.0.so.18(_ZN3JSC8JSObject13putInlineSlowEPNS_9ExecStateENS_12PropertyNameENS_7JSValueERNS_15PutPropertySlotE+0x2bf) [0x7fd112eeefbf]
28  0x7fd112ccde68 /usr/lib/libjavascriptcoregtk-4.0.so.18(+0x9efe68) [0x7fd112ccde68]
29  0x7fd112cb9992 /usr/lib/libjavascriptcoregtk-4.0.so.18(+0x9db992) [0x7fd112cb9992]
```

**Comment by killajoe** [comment]
https://forum.antergos.com/topic/9751/unable-to-install-cannot-create-download-package-list-metalinks

**Comment by karasu** [comment]
Fixed in Cnchi 0.14.425
